doc/eID/eid-flow-paula.md

```diff
@@ -21,13 +21,13 @@ sequenceDiagram
     browser ->> eService: access webapp
     eService ->> backend: start flow
     backend ->> server: start session
-    server -->> backend: return session identifier <br>[sessionId]
-    backend ->> backend: generate tcTokenURL
-    backend -->> eService: return <br>[tcTokenURL]
+    server -->> backend: return session identifier <br>[eIDSessionId]
+    backend ->> backend: generate <br>[tcTokenURL, useIDSessionId]
+    backend -->> eService: return <br>[tcTokenURL, useIDSessionId]
     eService ->> widget: integrate script which creates iframe <br>[tcTokenURL]
     widget ->> backend: (iframe) get widget page
-    backend -->> widget: return widget page
-    widget ->> widget: generate <br>[widgetSessionId, widgetSessionSecret]
+    backend -->> widget: return widget page <br>[widgetSessionId]
```

Comment:
I still can't overview the pro and cons of generating it on the server vs. generating it on the client. I think it's good like this, but wanted to mention my insecurity about it.